appservice/src/SiteWrapper.ts

```diff
@@ -17,6 +17,13 @@ import { ArgumentError } from './errors';
 import * as FileUtilities from './FileUtilities';
 import { localize } from './localize';
 
+const noSupport: string = localize('noSupport', 'This deployment source is not currently supported in VS Code and will be deployed by zip.');
```

Comment:
Where is this const used?

Can you add some screenshots of the experience?

Reply:
This was used for unsupported deployment sources when choosing a source type, but we have opted to remove them from the quickpick menu entirely.